lib/kontrast/page_test.rb

```diff
@@ -10,12 +10,21 @@ def initialize(prefix, name, path, headers: {}, url_params: {})
             super(prefix, name, path, headers)
             @width = prefix
             @url_params = url_params
+            @path = get_path_with_params(url_params)
+        end
+
+        def get_path_with_params(url_params)
+            uri = URI(@path)
```

Comment:
Using this instance variable here was a little bit confusing because I didn't know where it was coming from. Maybe just mentioning that it's already defined and we're re-defining it would be helpful.